app/models/subscribe_link.rb

```diff
@@ -0,0 +1,180 @@
+class SubscribeLink < ApplicationRecord
+  PLATFORMS = %w[apple spotify overcast pocketcasts youtube youtube_feed amazon antenna anytime apollo breez castamatic castbox castro curiocaster deezer fountain global goodpods gpodder hark iheart lnbeats luminary moon pandora player podbean podaddict podapp podguru_apple podguru_feed podrepublic podfriend podlp_apple podlp_guid podstation podurama podverse snipd sonnet steno truefans tunein].freeze
```

Comment:
i started with the platforms shown in the original design, which is why they are at the beginning, but the rest are in alphabetical order. if there are any changes to be made to this order or we want to define the most commonly chosen ones, let me know.